### DISCUSSION TITLE
Copy button

### DISCUSSION THREAD

**Discussion by Mage-Enderman** [I5UCC/VRCTextboxSTT#7] [Ideas]
As typing in VR is difficult even with good keyboard like XSOverlay, Oculus Link, Desktop+, etc
I think it'd be really good to have a button to copy the text from what you've said to be able to paste it to other places like Discord

**Comment by I5UCC** [comment]
Thank you for the feature idea!
Thats a great idea, ill probably implement it in two different ways:
- A copy button like you mentioned, that copies the last transcription at will.
- and automatic copying, it just writes to clipboard every time the transcription has finished.
The button will be the default, automatic will be a setting that people can switch to.

**Reply by Mage-Enderman** [reply]
Lovely!
Glad you think it's a good idea
I'm excited to see it implemented
You made a very useful program
